api-admin/src/main/java/bio/terra/pearl/api/admin/service/export/ExportIntegrationExtService.java

```diff
@@ -0,0 +1,73 @@
+package bio.terra.pearl.api.admin.service.export;
+
+import bio.terra.pearl.api.admin.service.auth.EnforcePortalStudyEnvPermission;
+import bio.terra.pearl.api.admin.service.auth.context.PortalStudyEnvAuthContext;
+import bio.terra.pearl.core.model.audit.ResponsibleEntity;
+import bio.terra.pearl.core.model.export.ExportIntegration;
+import bio.terra.pearl.core.model.export.ExportIntegrationJob;
+import bio.terra.pearl.core.service.exception.NotFoundException;
+import bio.terra.pearl.core.service.export.integration.ExportIntegrationJobService;
+import bio.terra.pearl.core.service.export.integration.ExportIntegrationService;
+import java.util.List;
+import java.util.UUID;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ExportIntegrationExtService {
+  private final ExportIntegrationService exportIntegrationService;
+
+  private final ExportIntegrationJobService exportIntegrationJobService;
+
+  public ExportIntegrationExtService(
+      ExportIntegrationService exportIntegrationService,
+      ExportIntegrationJobService exportIntegrationJobService) {
+    this.exportIntegrationService = exportIntegrationService;
+    this.exportIntegrationJobService = exportIntegrationJobService;
+  }
+
+  @EnforcePortalStudyEnvPermission(permission = "BASE")
```

Comment:
should all of these methods in this class be under the `export_integration` perm?